src/main/resources/db/migration/V1_3__create_table_melding.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE MELDING
+(
+    id                              SERIAL PRIMARY KEY,
+    uuid                            CHAR(36) NOT NULL UNIQUE,
+    created_at                      TIMESTAMPTZ NOT NULL,
+    innkommende                     BOOLEAN NOT NULL,
+    type                            VARCHAR(20) NOT NULL,
+    conversation                    VARCHAR(50) NOT NULL,
+    parent                          VARCHAR(50),
+    tidspunkt                       TIMESTAMPTZ NOT NULL,
+    arbeidstaker_personident        VARCHAR(11) NOT NULL,
+    behandler_personident           VARCHAR(11),
+    behandler_ref                   CHAR(36),
+    tekst                           TEXT,
+    antall_vedlegg                  INT NOT NULL
```

Comment:
Hva ble vi enige om rundt vedlegg? Hva gir det oss å vite antallet?

Reply:
Jeg antar at det er 0 eller 1 vedlegg på det vi sender ut, men for det som kommer inn kan det i prinsippet være mange vedlegg. Som vi har snakket om tidligere må de hentes fra padm2 for innkommende meldinger.